publish/views/asset.py

```diff
@@ -56,6 +57,11 @@ class AssetViewSet(NestedViewSetMixin, DetailSerializerMixin, ReadOnlyModelViewS
     filter_backends = [filters.DjangoFilterBackend]
     filterset_class = AssetFilter
 
+    @action(detail=True, methods=['GET'])
+    def download(self, request, **kwargs):
+        """Return a redirect to the file download in S3."""
```

Comment:
I'd say to remove S3 as it is an implementation detail and may not make sense for e.g. local deployments.

`"""Return a redirect to the file download in the object store."""`

Reply:
Good catch

Reply:
ea1227f